backend/routes/twoRandomToppings.ts

```diff
@@ -0,0 +1,33 @@
+import { Request, Response } from 'express'
+import expandToppingsCategories from '../expandToppingsCategories'
+import { getToppingsCollection } from '../db'
+import { SuccessResponse } from '../response'
+import { Topping, ToppingWithCategoryId } from '../topping'
+
+type TwoRandomToppingsResponse = SuccessResponse<[Topping,Topping]>
```

Comment:
what is 'type' doing?

Reply:
it's the type-level version of `const` basically, whenever you use `TwoRandomToppingsResponse` in a type, it's the same as using `SuccessResponse<[Topping,Topping]>`

Reply:
snazzy 💅